app-editors/gnome-latex/gnome-latex-3.42.0.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome2
+
+DESCRIPTION="Integrated LaTeX environment for GNOME"
+HOMEPAGE="https://wiki.gnome.org/Apps/GNOME-LaTeX https://gitlab.gnome.org/swilmet/gnome-latex"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~riscv ~x86"
+IUSE="+introspection +latexmk rubber"
+
+DEPEND="
```

Comment:
Would be nice to rearrange these dependencies to match the new `configure.ac` ordering:

````diff
-# Checks for dependencies
+# Dependencies
 AX_REQUIRE_DEFINED([PKG_CHECK_MODULES])
 PKG_CHECK_MODULES([DEP], [
-       glib-2.0 >= ${GLIB_REQUIRED_VERSION}
-       gio-2.0 >= ${GLIB_REQUIRED_VERSION}
-       gtk+-3.0 >= ${GTK_REQUIRED_VERSION}
-       gtksourceview-4 >= ${GTKSOURCEVIEW_REQUIRED_VERSION}
-       amtk-5 >= ${AMTK_REQUIRED_VERSION}
-       tepl-6 >= ${TEPL_REQUIRED_VERSION}
-       gspell-1 >= ${GSPELL_REQUIRED_VERSION}
-       gee-0.8 >= ${GEE_REQUIRED_VERSION}
+       gio-2.0 >= 2.56
+       gtk+-3.0 >= 3.22
+       gspell-1 >= 1.8
+       amtk-5 >= 5.6
+       gtksourceview-4
+       tepl-6 >= 6.2
+       gee-0.8 >= 0.10
        gsettings-desktop-schemas
 ])
````